### PR TITLE
calculate square root of number

### PR DESCRIPTION
**Pull request by fneu**

### CODE REVIEW

File: change.py [hunk 1]
```diff
@@ -0,0 +1,6 @@
+import math
+import this
```
Comment (fneu) [inline change.py]: this import is unneccessary

Comment (fneu) [inline change.py]: unneeded import

File: change.py [hunk 2]
```diff
@@ -0,0 +1,6 @@
...
+
+def square_root(number):
+	sr = math.sqrt(number)
+        return sr
```
Comment (fneu) [inline change.py]: inconsistent whitespace

**Comment by sils1297** [comment]
@fneu what are you doing?

**Pull request closed without merging**